src/packages/utils/error.ts

```diff
@@ -0,0 +1,21 @@
+// Clicar no botão para criar uma eleição sem dados - Error 1
+
+// Clicar no botão de iniciar eleição sem selecionar eleição - Error 2
```

Comment:
You can remove these comments here as these errors are generic and don't need to relate to a specific part of the system